### PR TITLE
Add LT and GT to supported otn chars

### PR DESCRIPTION
**Pull request by ChrisLovering**
Does what it says on the tin.
https://github.com/python-discord/site/pull/595 must be merged first

### CODE REVIEW

File: bot/converters.py [hunk 1]
```diff
@@ -402,9 +402,9 @@ def translate_name(cls, name: str, *, from_unicode: bool = True) -> str:
         If `from_unicode` is True, the name is translated from a discord-safe format, back to normalized text.
         """
         if from_unicode:
-            table = str.maketrans(cls.ALLOWED_CHARACTERS, '𝖠𝖡𝖢𝖣𝖤𝖥𝖦𝖧𝖨𝖩𝖪𝖫𝖬𝖭𝖮𝖯𝖰𝖱𝖲𝖳𝖴𝖵𝖶𝖷𝖸𝖹ǃ？’’-')
+            table = str.maketrans(cls.ALLOWED_CHARACTERS, '𝖠𝖡𝖢𝖣𝖤𝖥𝖦𝖧𝖨𝖩𝖪𝖫𝖬𝖭𝖮𝖯𝖰𝖱𝖲𝖳𝖴𝖵𝖶𝖷𝖸𝖹ǃ？’’-＜＞')
```
Comment (onerandomusername) [inline bot/converters.py:405]: Should this second string be a constant since its in two places?

Comment (ChrisLovering) [reply bot/converters.py]: Since I'm here, why not. Force pushed

**Review by onerandomusername — APPROVED** [review]
LGTM

**Review by Akarys42 — APPROVED** [review]
lgtm

**Review by D0rs4n — APPROVED** [review]

**Pull request merged**